scripts/deploy

```diff
@@ -0,0 +1,163 @@
+  
+  if [ ${K3S_FLAG} -eq 1 ] ; then
+      time timeout 300 bash -c 'while ! (k3s kubectl --namespace minio rollout status --timeout 10s deploy/minio 2>/dev/null); do sleep 5; done'
+  fi
+
+  ${KUBECTL_CMD} create secret generic miniocreds --from-literal=accessKey=inspectorgadget --from-literal=secretKey=gogadgetgo
+}
+
+deploy_backup_restore() {
+  if [ -z "${KUBECONFIG}" ]; then
+   echo "Please set your Kubeconfig environment variable before running."
+   exit 0
+  fi
+
+  . ./scripts/version &> /dev/null
+
+  helm repo add rancher-chart https://charts.rancher.io
+  helm install --wait --create-namespace -n cattle-resources-system rancher-backup-crd dist/artifacts/rancher-backup-crd-${HELM_VERSION_DEV}.tgz
+  helm install --wait -n cattle-resources-system rancher-backup dist/artifacts/rancher-backup-${HELM_VERSION_DEV}.tgz \
+      --set image.repository=${REPO}/backup-restore-operator \
+      --set image.tag=${TAG} \
+      --set debug=true 
+}
+
+create_backup() {
+  if [ -z "${KUBECONFIG}" ]; then
+   echo "Please set your Kubeconfig environment variable before running."
+   exit 0
+  fi
+
+  KUBECTL_CMD="kubectl"
+
+  BASE64CA=$(cat public.crt |base64)
+
+  if command -v k3s &> /dev/null ; then
+      KUBECTL_CMD="k3s kubectl"
+      BASE64CA=$(cat public.crt |base64 -w0)
+  fi
+
+  cat <<EOF > backup.yaml
+apiVersion: resources.cattle.io/v1
+kind: Backup
+metadata:
+  name: s3-recurring-backup
+spec:
+  storageLocation:
+    s3:
+      credentialSecretName: miniocreds
+      credentialSecretNamespace: default
+      bucketName: rancherbackups
+      endpoint: minio.minio.svc.cluster.local:9000
+      endpointCA: ${BASE64CA}
+  resourceSetName: rancher-resource-set
+  schedule: "@every 30s"
+  retentionCount: 2
+EOF
+
+  ${KUBECTL_CMD} create -f backup.yaml
+}
+
+uninstall_charts() {
+  if [ -z "${KUBECONFIG}" ]; then
+   echo "Please set your Kubeconfig environment variable before running."
+   exit 0
+  fi
+  helm uninstall -n minio minio
+  helm uninstall -n cattle-resources-system rancher-backup 
+  helm uninstall -n cattle-resources-system rancher-backup-crd
+  kubectl delete deploy rancher-backup -n cattle-resources-system
+}
+
+
+case $1 in
+    template)
+         helm template rancher-backup-0.0.0-dev.tgz --output-dir=./bla/ --set image.tag=${TAG} --set imagePullPolicy=IfNotPresent
+    ;;
+
+    backup-restore) 
+        if [! -f "public.crt" || ! -f "private.key"]; then
+            create_cert
+        fi
+        deploy_backup_restore
+    ;;
+
+    minio)
+        create_cert
+        deploy_minio
+    ;;
+
+    backup)
+        create_backup
+    ;;
+
+    remove_charts)
+        uninstall_charts
+    ;;
+
+    clean)
+        rm private.key public.crt backup.yaml
+    ;;
+
+    *)
+        echo "./deploy [template/build/backup-restore/minio//remove-charts/clean]"
+        echo ""
+        echo "The following variable need to be exported"
```

Comment:
This is also shown if its already exported, we should change it to `KUBECONFIG is pointing to $KUBECONFIG` if its already set to not confuse users.